doc/docs/deploy-manage/deploy/environment-variables.md

```diff
@@ -0,0 +1,166 @@
+# Configure Environment Variables
+
+When you use Pachyderm, you can define environment variables that
+can transmit the required configuration directly to your application.
+
+In Pachyderm, you can define the following types of environment
+variables:
+
+* `pachd` environment variables that define parameters for your
+Pachyderm daemon container.
+
+* Pachyderm worker environment variables that define parameters
+on the Kubernetes pods that run your pipeline code.
+
+You can configure environment variables as you typically do
+in a Linux environment. For example, to configure a `PACH_JOB_ID`
```

Comment:
FWIW, `PACH_JOB_ID` isn't something that users typically set; rather it's something that they typically read after Pachyderm sets it. For example, if you're writing a pipeline and you want to know ID of the job your code is currently running in (e.g. because you're writing data to an external system and you want to be able to tie it back to the Pachyderm job) then your code could discover that information by reading `PACH_JOB_ID`.